frontend/src/styles/vars.scss

```diff
@@ -12,7 +12,7 @@ $screens: (
     'xxl': $xxl,
 );
 
-$spaces: 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20;
+$spaces: 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 120;
```

Comment:
nit: I'd rather have this as a var in `vars.scss` and keep this 1:1 with tailwind as much as possible

Reply:
I don't think this deviates too much from tailwind since it's another width value, and it would be out of place as a separate variable. Tailwind goes up to w-96 which means we should technically have values up to 96*4=384 to be 1:1 w/ tailwind.

Reply:
I think @benjackwhite wanted to limit these to not overload the automatic creation of classes. He's the expert here, I'll let him confirm this is fine 😅